### PR TITLE
Add meeting agenda 2024-02-21

### PR DESCRIPTION
**Pull request by VirginiaBalseiro**
Request to merge in 24 hours following meeting.

### CODE REVIEW

File: meetings/2024-02-21.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+
+### Co-chairs Rotation Schedule
+URL: https://github.com/solid/specification/discussions/618
+
+
+### Solid Symposium 2024
+
+* RG: I shall be organizing the [Real-Time Solid session](https://cxres.inrupt.net/public/SoSy24/RealTimeSolid/) at [Solid Symposium 2024](https://events.vito.be/sosy2024) to be held 2-3 May 2024 in Leuven, Belgium.
+* RG: Anyone interested in speaking on notifications-related issues or intersted in putting up a poster on the topic, please get in touch with me.
```
Comment (csarven) [inline meetings/2024-02-21.md:67]:
```suggestion
* RG: Anyone interested in speaking on notifications-related issues or interested in putting up a poster on the topic, please get in touch with me.
```

File: meetings/2024-02-21.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+URL: https://github.com/solid/specification/pull/598
+
+* VB: From last week:
+
+> ACTION: eP to make PR with security threat use cases and capture one discussed in the PR
+
+* VB: Updates?
+* eP: https://github.com/solid/specification/pull/625
+* VB: Would anyone like to review?
+* SC: The question is: is this a work item? Reviewing is good, but if the group is committing to it as a work item, follow normal procedure. Good start for best practices, my only comment is what is the soruce, what constitutes a best practice? For each of the points, it'd be good to have a source as to why the community/group.. who is claiming this is a good practice as opposed to a decision by a particular implementation.
```
Comment (csarven) [inline meetings/2024-02-21.md:82]:
```suggestion
* SC: The question is: is this a work item? Reviewing is good, but if the group is committing to it as a work item, follow normal procedure. Good start for best practices, my only comment is what is the source, what constitutes a best practice? For each of the points, it'd be good to have a source as to why the community/group.. who is claiming this is a good practice as opposed to a decision by a particular implementation. 
```

File: meetings/2024-02-21.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+* eP: https://github.com/solid/specification/pull/625
+* VB: Would anyone like to review?
+* SC: The question is: is this a work item? Reviewing is good, but if the group is committing to it as a work item, follow normal procedure. Good start for best practices, my only comment is what is the soruce, what constitutes a best practice? For each of the points, it'd be good to have a source as to why the community/group.. who is claiming this is a good practice as opposed to a decision by a particular implementation. 
+* eP: This would be a non-normative document. Implementations would allow configuration options for their cases. For security vulnerabilities, we should know about it and provide them. We could rename if BP is not most accurate. Most important to have it in one place. Information as best of our capability and to make informed choices.
+* AC: In the OAuth world, there are people that implement OAuth based systems and may not be aware of all the details of security concerns that exists in that space. This kind of a document that would help implementers. To SC's point on concensus, it'd be good to have this info available, ultimately it'd make Solid safer and implementer.
```
Comment (csarven) [inline meetings/2024-02-21.md:84]:
```suggestion
* AC: In the OAuth world, there are people that implement OAuth based systems and may not be aware of all the details of security concerns that exists in that space. This kind of a document that would help implementers. To SC's point on consensus, it'd be good to have this info available, ultimately it'd make Solid safer and implementer.
```

File: meetings/2024-02-21.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+ACTION: SC to put comments on PR.
+
+### Should Solid (storage) servers support "RDF documents" containing multiple subjects (or quads)?
+URL: https://github.com/solid/specification/issues/610
+* VB: From last week:
+>  PROPOSAL: attach label and ping Maxime to resolve it
+* VB: Update?
+* SC: There's plenty of feedback. I initially suggested to label this and have Maxime respond. AFAIK the questions were addressed. There's no more to do on that issue unless there's more information coming up. We can leave it there until Maxime responds. This does not require any change in the spec unless Maxime (or others) consider the response is not adequate. 
+* eP: I'd suggest to close issue and reopen if need to.
+* TT: Concerned by the title of the issue and perhaps it should've been changed. Haven't seen a suggestion where a Solid server shold only contain a single subject. The issue in and itself on how the world works, including the misunderstandings in the issue. By all means, be gentle but close with no action.
```
Comment (csarven) [inline meetings/2024-02-21.md:96]:
```suggestion
* TT: Concerned by the title of the issue and perhaps it should've been changed. Haven't seen a suggestion where a Solid server should only contain a single subject. The issue in and itself on how the world works, including the misunderstandings in the issue. By all means, be gentle but close with no action.
```

File: meetings/2024-02-21.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+### Fine tune wording of the requirement for including Content-Type in the response
+URL: https://github.com/solid/specification/issues/565
+
+* eP: HTTP spec only has SHOULD and SP elevates to MUST. Discussion was about awkward terminology in the HTTP spec.
+* SC: Need to jog my memory / revisit issue. What's the question?
+* RG: Since HEAD is low resource intensive call, Wouter's argument is that you don't need to include Content-Type.
+* VB: Let's revisit this next week. 
+* TT: What headers can HEAD omit? My understanding was same as GET without body.
+* AC: My understanding in RFC 9110, generally yes, but there are few headers like Content-Length being one where a few can be ommitted. I think Content-Type is one of those that can be ommitted.
```
Comment (csarven) [inline meetings/2024-02-21.md:110]:
```suggestion
* AC: My understanding in RFC 9110, generally yes, but there are few headers like Content-Length being one where a few can be omitted. I think Content-Type is one of those that can be omitted.
```

File: meetings/2024-02-21.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+* RG: From [9110](https://www.rfc-editor.org/rfc/rfc9110#section-9.3.2):
+  > The server SHOULD send the same header fields in response to a HEAD request as it would have sent if the request method had been GET. However, a server MAY omit header fields for which a value is determined only while generating the content. For example, some servers buffer a dynamic response to GET until a minimum amount of data is generated so that they can more efficiently delimit small responses or make late decisions with regard to content selection. Such a response to GET might contain Content-Length and Vary fields, for example, that are not generated within a HEAD response. These minor inconsistencies are considered preferable to generating and discarding the content for a HEAD request, since HEAD is usually requested for the sake of efficiency. 
+* SC: Determining resource type like multimedia, RDF source, or something else. If a server does not do that, client needs to do a GET. Do we want to force clients to do that? I would lean towards making sure the server can generate Contant Type for HEAD. Lowers total number of requests.
```
Comment (csarven) [inline meetings/2024-02-21.md:113]:
```suggestion
* SC: Determining resource type like multimedia, RDF source, or something else. If a server does not do that, client needs to do a GET. Do we want to force clients to do that? I would lean towards making sure the server can generate Content-Type for HEAD. Lowers total number of requests.
```

File: meetings/2024-02-21.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+* AC: +1 (we want to encourage the use of HEAD).
+* eP: Agree, for example when an application wants to display differen icons. If resource is created in ways other than Solid Protocol.
```
Comment (csarven) [inline meetings/2024-02-21.md:115]:
```suggestion
* eP: Agree, for example when an application wants to display different icons. If resource is created in ways other than Solid Protocol.
```

**Review by csarven — COMMENTED** [review]
Just spellchecked. I tend to spellcheck before PRing so that I get all the fake internet points and less points for Ted...

**Review by TallTed — CHANGES_REQUESTED** [review]
Minor, grammar, etc.

**Pull request merged**